csun-metalab/nomi
# HOTFIX-11-27-2018--fixUploadResponse

# Description
Fixed the upload response message that appears after uploading and confirming a photo.
  
# Testing
1. Visit any student and click the edit button on their photo.
2. Upload a photo and confirm the image.
3. Ensure the upload message is now set to the proper theme color.
4. Visit a student profile and redo step 2; ensure the response message is now at the top (right beneath the top nav-bar).
5. Ensure the response isn't clipping out of place at different screen resolutions.
